src/PackageDependencyViewExtension/PackageDependencyViewExtension.cs

```diff
@@ -69,7 +70,8 @@ public void Startup(ViewStartupParams sp)
 
         public void Loaded(ViewLoadedParams p)
         {
-            var packageDependencyView = new PackageDependencyView();
+            var dynamoViewModel = p.DynamoWindow.DataContext as DynamoViewModel;
```

Comment:
I know other extensions do this - but it's not ideal - we should try to avoid it or provide a clearer, smaller, more consistent API if we can - that we will be able to maintain long term.

Reply:
Agree